rest/src/routes/finalizationRoutes.js

```diff
@@ -35,18 +38,31 @@ module.exports = {
 		const { connections } = services;
 		const { timeout } = services.config.apiNode;
 
+		const readAndParseNetworkPropertiesFile = () => {
+			const readFile = util.promisify(fs.readFile);
+			try {
+				return readFile(services.config.network.propertiesFilePath, 'utf8')
+					.then(fileData => ini.parse(fileData));
+			} catch (err) {
+				// If API node is not updated, then use static fork height
+				return new Promise(resolve => resolve({ fork_heights: { votingKeyLinkV2: '215500' } }));
```

Comment:
about the hardcoded value, is this for testnet node not updated? In the future, votingKeyLinkV2 will not be provided for testnet/publicnet with the "main" server

Reply:
yeah, this is only for those synced with outdated server code.